test/libyul/yulOptimizerTests/unusedStoreEliminator/function_end.yul

```diff
@@ -0,0 +1,20 @@
+{
+    function f() {
+        let x := calldataload(2)
+        sstore(x, 2)
+        // This cannot be removed because we do not know what happens after the function.
+        sstore(x, 3)
```

Comment:
Doesn't this make more sense with ``mstore``s? The second ``sstore`` couldn't be removed even if it wasn't a function...